packages/types/src/api/web/global/scim/users.ts

```diff
@@ -0,0 +1,83 @@
+export interface ScimUser {
+  schemas: ["urn:ietf:params:scim:schemas:core:2.0:User"]
+  id: string
+  externalId: string
+  meta: {
+    resourceType: "User"
+    created: string
+    lastModified: string
+  }
+  userName: string
+  name: {
+    formatted: string
+    familyName: string
+    givenName: string
+  }
+  active: boolean
+  emails: [
+    {
+      value: string
+      type: "work"
+      primary: true
+    }
+  ]
+}
+
+export interface ScimListResponse {
+  schemas: ["urn:ietf:params:scim:api:messages:2.0:ListResponse"]
+  totalResults: number
+  Resources: ScimUser[]
+  startIndex: number
+  itemsPerPage: number
+}
+
+export interface ScimUserRequest {
```

Comment:
Could we add the verb for this request, like `ScimCreateUserRequest`, or if it's the same for create and update could go with `ScimSaveUserRequest`. 

For `ScimListResponse`, given we have users and groups this could conflict, the adding in user might help: `ScimUserListResponse`

Reply:
Totally agree, I actually changed that in a further PR. Changing it here

Reply:
Same, already modifies in a further PR. Changing it here